ext/standard/array.c

```diff
@@ -4444,7 +4444,8 @@ PHP_FUNCTION(array_flip)
 			}
 			zend_symtable_update(Z_ARRVAL_P(return_value), Z_STR_P(entry), &data);
 		} else {
-			php_error_docref(NULL, E_WARNING, "Can only flip STRING and INTEGER values!");
+			zend_type_error("Can only flip STRING and INTEGER values!");
```

Comment:
We should also drop the ALLCAPS and the exclamation mark ... this is a very aggressive warning ;)

Reply:
@nikic I actually noticed that too and there are more exclamation marks which can be removed. I think a separate PR is better

Reply:
Should I do the same for array_count_values and group both of them in a single PR ?